packages/task-console/package.json

```
@@ -31,7 +31,11 @@
     "cypress:open": "cypress open"
   },
   "dependencies": {
-    "@kogito-apps/common": "1.0.0"
+    "@kogito-apps/common": "1.0.0",
+    "ajv": "^6.10.2",
```

@pefernan, we've established to not use version ranges ( ^ ) as in Jenkins the actually build uses a different npm registry, that can cause different version to be resolved.
```suggestion
    "ajv": "6.10.2",
```

---

@cristianonicolai sure thanks!